test/async-response-transform.test.js

```diff
@@ -1,32 +1,32 @@
-import test from 'ava'
 import { create } from '../lib/apisauce'
 import createServer from './_server'
 import getFreePort from './_getFreePort'
+import { beforeAll, afterAll, test } from '@jest/globals'
```

Comment:
nit: If these are configured as globals, you shouldn't need to import them directly, right?

Looks like it added them only in some files but not all. I'm find with explicit imports if we want to do that, but should probably be consistent.

Also, it looks like it didn't explicitly import all of the globals? Like `expect` is not imported, but I saw that it was in the other files.